client/src/ui/PageFooter.tsx

```diff
@@ -0,0 +1,172 @@
+Except for the Login Page and Sign Up Page
+*/
+
+export const PageFooter = () => {
+  return (
+    <footer
+      style={{
+        backgroundColor: "#f9f9fc",
+        padding: "20px 40px",
+        display: "flex",
+        justifyContent: "space-evenly",
+        flexWrap: "wrap",
+      }}
+    >
+      {/* Left Section */}
+      <div
+        style={{
+          display: "flex",
+          flexWrap: "nowrap",
+        }}
+      >
+        <Link
+          href="#"
+          style={{ margin: "0 40px", color: "#000", textDecoration: "none" }}
+        >
+          <Typography>Explore</Typography>
+        </Link>
+        <Link
+          href="#"
+          style={{ margin: "0 40px", color: "#000", textDecoration: "none" }}
+        >
+          <Typography>About Us</Typography>
+        </Link>
+        <Link
+          href="#"
+          style={{ margin: "0 40px", color: "#000", textDecoration: "none" }}
+        >
+          <Typography>Contact</Typography>
+        </Link>
+      </div>
+
+      {/* Center Section */}
+      <div
+        style={{
+          display: "flex",
+          alignItems: "center",
+          textAlign: "center",
+          position: "relative",
+          right: "15px",
+        }}
+      >
+        <Divider
+          style={{
+            flexGrow: 1,
+            backgroundColor: "#ccc",
+            width: "20px",
+            height: "1px",
+            margin: "0 20px",
+          }}
+        />
+        <Typography
+          variant="h4"
+          style={{
+            fontWeight: "bold",
+            fontSize: "1.25rem",
+            textAlign: "center",
+          }}
+        >
+          SqueakPeek
+        </Typography>
+        <Divider
+          style={{
+            flexGrow: 1,
+            backgroundColor: "#ccc",
+            width: "20px",
+            height: "1px",
+            margin: "0 20px",
+          }}
+        />
+      </div>
+
+      {/* Right Section */}
+      <div
+        style={{
+          display: "flex",
+          alignItems: "center",
+        }}
+      >
+        <Link href="#">
+          <Image
+            src="/x-logo/X-logo-black.png"
+            alt="X"
+            width={24}
+            height={24}
+            style={{
+              margin: "0 40px",
+              opacity: 0.7,
+              transition: "opacity 0.3s ease",
+            }}
+            onMouseEnter={(e) => (e.currentTarget.style.opacity = "1")}
+            onMouseLeave={(e) => (e.currentTarget.style.opacity = "0.7")}
+          />
+        </Link>
+        <Link href="#">
+          <Image
+            src="/Instagram-logo/Instagram_Glyph_Black.svg"
+            alt="Instagram"
+            width={24}
+            height={24}
+            style={{
+              margin: "0 40px",
+              opacity: 0.7,
+              transition: "opacity 0.3s ease",
+            }}
+            onMouseEnter={(e) => (e.currentTarget.style.opacity = "1")}
```

Comment:
(nit) - Instead of applying this logic, you can just the Button component that from MUI that does this for you: https://mui.com/material-ui/react-button/

![image](https://github.com/user-attachments/assets/901219a1-37f7-4581-a50b-d79b7bcc5799)